lapis/job.py

```diff
@@ -117,6 +87,7 @@ def __repr__(self):
 
 
 async def job_to_queue_scheduler(job_generator, job_queue, **kwargs):
+    from lapis.utility.monitor import sampling_required
```

Comment:
Is there a reason why this is imported inside the function, not at the top of the module?

Reply:
Not anymore, thanks.